### PR TITLE
Feat(eos_designs): Support for Ipv6 network services

### PR DESCRIPTION
**Pull request by onurgashi**
## Change Summary

This PR will add IPv6 support to tenants structure.

```
 < tenant_a >
    vrfs:
      < tenant_a_vrf_1 >:
        svis:
          < 1-4096 >:
            ipv6_address_virtual: < IPv6_address/Mask >
            ipv6_address_virtual_secondaries:
              - < IPv6_address/Mask >
            ipv6_virtual_router_addresses:
              - < IPv6_address/Mask | IPv6_address >
            nodes:
              < l3_leaf_inventory_hostname_1 >:
                ipv6_address: < IPv6_address/Mask >   
        ipv6_static_routes:
          - destination_address_prefix: < IPv6_address/Mask >
            gateway: < IPv6_address >
            distance: < 1-255 >
            tag: < 0-4294967295 >
            name: < description >
            metric: < 0-4294967295 >
            interface: < interface >
            nodes: [ < node_1 >, < node_2 >]
```

## Related Issue(s)

Depends on VARPv6 PR for cli_config_gen role

## Component(s) name

`arista.avd.eos_designs`

## How to test

Molecule `eos_designs_unit_tests`

### User Checklist

- N/A

### Repository Checklist

- [x] My code has been rebased from devel before I start
- [x] I have read the [**CONTRIBUTING**](https://avd.sh/en/latest/docs/contribution/overview.html) document.
- [x] My change requires a change to the documentation and documentation have been updated accordingly.
- [x] I have updated [molecule CI](https://github.com/aristanetworks/ansible-avd/tree/devel/ansible_collections/arista/avd/molecule) testing accordingly. (check the box if not applicable)

### CODE REVIEW

File: ansible_collections/arista/avd/molecule/eos_designs_unit_tests/documentation/devices/DC1-BL1A.md
Comment (gmuloc) [inline ansible_collections/arista/avd/molecule/eos_designs_unit_tests/documentation/devices/DC1-BL1A.md]: is it expected that all the documentation for devices is gone?

Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/molecule/eos_designs_unit_tests/documentation/devices/DC1-BL1A.md]: fixed

**Comment by tgodaA** [comment]
Sorry @onurgashi , I think I messed up your branch. I didn't realize that your branch wasn't rebase off the devel by me. So all of the old commits ended up in your branch.

**Review by ClausHolbechArista — APPROVED** [review]
LGTM

File: ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_FABRIC.yml
Comment (jonxstill) [inline ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_FABRIC.yml]: It's not a problem now because you've already had the reviews necessary - but adding the `wan` tag here vastly increases the blast radius (because there's changes to other tenants) of your change on the molecule artefacts, meaning there's a lot more work for people to do to review the change (or rather a lot more cruft for people to wade through).  This could have been a `wanv6` tag for example that would have only affected Tenant_D and a more limited number of devices.

Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_FABRIC.yml]: +1 this annoyed me too, but since he was in a hurry...

Comment (ClausHolbechArista) [reply ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_FABRIC.yml]: BTW note we need 2 approvals on ansible-avd repo

File: ansible_collections/arista/avd/roles/eos_designs/doc/network-services.md
Comment (jonxstill) [inline ansible_collections/arista/avd/roles/eos_designs/doc/network-services.md:254]: Is there a check that verifies that the ipv6 interface address has been specified if we try to use a VARPv6 address?

**Review by jonxstill — APPROVED** [review]
LGTM - I can see my comment about IPv6 VARP applies for IPv4 too, so disregard that!

File: ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_FABRIC.yml
Comment (jonxstill) [reply ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DC1_FABRIC.yml]: Yep, just added a 2nd.

**Review by ClausHolbechArista — CHANGES_REQUESTED** [review]
I will block this from merging and fix the molecule to be more precise.

**Review by ClausHolbechArista — APPROVED** [review]
LGTM

**Pull request merged**
